src/sentry_config.py

```diff
@@ -0,0 +1,30 @@
+from configparser import ConfigParser
+
+import requests
+
+SENTRY_CONFIG_API_URL = (
+    "https://api.github.com/repos/{owner}/.sentry/contents/sentry_config.ini"
+)
+SENTRY_CONFIG_RAW_URL = (
+    "https://raw.githubusercontent.com/{owner}/.sentry/main/sentry_config.ini"
+)
+
+
+def fetch_dsn_for_github_org(org: str) -> str:
```

Comment:
they'll need to add this special `.sentry` repo to the installation as well -- and we'll need to make this an api request to get the contents of that file (that way we can access it if it's private)

here's an example from pre-commit.ci reading a single file from a repository:

```python
def api_get_repo_file(
        repo_id: int,
        path: str,
        access_token: str,
        *,
        ref: str | None = None,
) -> str:
    if ref is not None:
        query = f'?ref={urllib.parse.quote(ref)}'
    else:
        query = ''

    resp = req(
        f'https://api.github.com/repositories/{repo_id}/contents/{path}{query}',  # noqa: E501
        headers=at_headers(access_token),
    )

    if not isinstance(resp.json, dict):
        raise NotAFileError('directory')
    elif resp.json['type'] != 'file':
        raise NotAFileError(resp.json['type'])

    assert resp.json['encoding'] == 'base64', resp.json['encoding']
    contents = base64.b64decode(resp.json['content']).decode()
    return contents
```

we'll probably want to cache this value as well if we're likely to get a lot of requests to this